osm2streets/src/transform/mod.rs

```diff
@@ -7,59 +7,33 @@ mod collapse_short_road;
 mod dual_carriageways;
 mod parallel_sidepaths;
 mod remove_disconnected;
-mod sausage_links;
 
 /// An in-place transformation of a `StreetNetwork`.
 pub enum Transformation {
     ZipSidepaths,
     RemoveDisconnectedRoads,
     CollapseShortRoads,
     CollapseDegenerateIntersections,
-    CollapseSausageLinks,
     MergeDualCarriageways,
 }
 
 impl Transformation {
     /// Useful for test cases and small clipped areas. Doesn't remove disconnected roads.
     pub fn standard_for_clipped_areas() -> Vec<Self> {
         vec![
-            Transformation::CollapseSausageLinks,
             Transformation::CollapseShortRoads,
             Transformation::CollapseDegenerateIntersections,
             // The above may discover more roads to collapse
             Transformation::CollapseShortRoads,
         ]
     }
 
-    /// A full suite of transformations for A/B Street.
```

Comment:
I don't plan on updating A/B Street anytime remotely soon. This code could move there if needed.